File: 153-FindMinimumInRotatedSortedArray/153-FindMinimumInRotatedSortedArray.js
// Last updated: 10/03/2026, 10:03:38
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMin = function(arr) {
    let min=arr[0];
    for(let i=0;i<arr.length;i++){
        if(arr[i]<min){
            min=arr[i];
        }
    }
    return min;
};

File: 153-FindMinimumInRotatedSortedArray/153-FindMinimumInRotatedSortedArray.cpp
// Last updated: 03/08/2026, 10:13:12
class Solution {
public:
    int findMin(vector<int>& nums) {
        int mn=nums[0];
        for(int i=0;i<nums.size();i++){
            if(nums[i]<mn){
                mn=nums[i];
            }
        }
        return mn;
    }
};